doc/how_to/migrate/from_streamlit.md

```diff
@@ -0,0 +1,950 @@
+
+### Input Migration Steps
+
+You should
+
+- replace your Streamlit `st.some_widget` function with the corresponding Panel
+`pn.widgets.SomeWidget` class.
+
+You can identify the corresponding widget via the
+[Widgets Section](https://panel.holoviz.org/reference/index.html#widgets) of the
+[Component Gallery](https://panel.holoviz.org/reference/index.html).
+
+## Layouts
+
+*Layouts* helps you organize your *panes* and *widgets*.
+
+Panel provides layouts similar to the ones you know from Streamlit and some unique ones.
+
+Check out the [Layouts Section](https://panel.holoviz.org/reference/index.html#layouts) of the
+[Component Gallery](https://panel.holoviz.org/reference/index.html) for the full list of *layouts*.
+
+### Streamlit Layout Example
+
+```python
+import streamlit as st
+
+col1, col2 = st.columns(2)
+
+with col1:
+    st.image("https://streamlit.io/images/brand/streamlit-logo-primary-colormark-darktext.png")
+    st.write("# A faster way to build and share data apps")
+
+with col2:
+    st.image("https://panel.holoviz.org/_images/logo_horizontal_light_theme.png")
+    st.write("# The powerful data exploration & web app framework for Python")
+```
+
+The app looks like
+
+![Streamlit Layout Example](https://user-images.githubusercontent.com/42288570/243353648-add1e7af-26ba-428b-ba47-2f602ff3ce93.png)
+
+I would love to align the images (and texts) but I could not find functionality like *row*, *margin* or *spacing* to do this.
+
+### Panel Layout Example
+
+You will find Panels *layouts* in the `pn` module.
+
+```python
+import panel as pn
+
+pn.extension(sizing_mode="stretch_width", template="bootstrap")
+
+col1 = pn.Column(
+    pn.pane.Image("https://streamlit.io/images/brand/streamlit-logo-primary-colormark-darktext.png"),
+    pn.panel("# A faster way to build and share data apps")
+)
+col2 = pn.Column(
+    pn.pane.Image("https://panel.holoviz.org/_images/logo_horizontal_light_theme.png"),
+    pn.panel("# The powerful data exploration & web app framework for Python"),
+)
+
+pn.Row(col1, col2).servable()
+```
+
+Panels `Column` and `Row` are *list like* objects. So you can use familiar methods like `.append`,
+`.pop` and `[]` indexing when you work with them. For the details check out the
+[`Column` Guide](https://panel.holoviz.org/reference/layouts/Column.html) and
+the [`Row` Guide](https://panel.holoviz.org/reference/layouts/Row.html).
+
+The app looks like
+
+![Panel Layout Example](https://user-images.githubusercontent.com/42288570/243362603-45ba78a4-d67b-43bc-b3c2-386105fe6ed8.png)
+
+### Layout Migration Steps
+
+You should
+
+- replace your Streamlit `st.some_layout` function with the corresponding Panel
+`pn.SomeLayout` class.
+
+You can identify the relevant layout to migrate to in the
+[Layouts Section](https://panel.holoviz.org/reference/index.html#layouts) of the
+[Component Gallery](https://panel.holoviz.org/reference/index.html).
+
```

Comment:
I think we should offer some insight that Panel has a lot more kinds of layouts than Streamlit like FlexBox, GridSpec, etc.

Reply:
Yes. But would be something for the comparison background article.

Reply:
I think a brief mention won't hurt.

Reply:
Feel free to contribute an improvement 😄